pwndbg/pdb.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+This module serves as utility for the developers to get pdb debugging session right, whenever standard:
+    import pdb; pdb.set_trace()
+Can't do its job properly (that happens often as gdb does some magic with stdout/stderr).
+
+Thanks to Dmoreno: http://stackoverflow.com/questions/17074177/how-to-debug-python-cli-that-takes-stdin
+
+Usage:
+    import pwndbg.pdb
+
+Importing the module will print out pretty message on what to do next.
+"""
+import functools
+import pdb
```

Comment:
> Can we use it to prevent warnings

No, the warnings aren't emitted from Python.

> Does it mean that current implementation has problems

Probably